DataBiosphere/Saturn-UI
# [WM-2108] Workflows App Mixpanel

### Jira Ticket: https://broadworkbench.atlassian.net/browse/WM-2108




## Summary of changes:


### What
- Adding an event to track workflow launches in the new workflows app

### Why
- We want to measure workflow launch metrics

### Testing strategy
- Turn on Azure workflows tab feature preview
- Submit workflow(s) and check [Mixpanel](https://mixpanel.com/s/3Zp40S) or [bard logs](https://console.cloud.google.com/logs/query?project=terra-bard-dev) for new event

### Visual Aids
Mixpanel Dev event:
<img width="1175" alt="Screenshot 2023-07-20 at 10 27 43 AM" src="https://github.com/DataBiosphere/terra-ui/assets/67511512/322b598e-ba54-46ab-9779-4f5c3827cc0e">
